examples/simulate-proposal/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "example-simulate-proposal",
+  "version": "1.37.0-rc.2",
+  "private": true,
+  "main": "index.js",
+  "author": "Nami Shah <nami@openzeppelin.com>",
+  "license": "MIT",
+  "scripts": {
+    "start": "node index.js"
+  },
+  "dependencies": {
+    "defender-admin-client": "1.37.0-rc.2",
```

Comment:
I think we should match this one with the version of the package. Can you double-check?

It's updated when the version is bumped via lerna and I'm afraid of this getting broken because not being synced with all of the other packages

Reply:
The current package is the same version.